src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -31,8 +33,31 @@ private static void setupPresetTests(Application application) {
 		SelectTestFigure2OptionListener selectTestFigure2OptionListener = new SelectTestFigure2OptionListener(
 				DriverFeature.getDriverManager());
 
+
+		ComplexCommandFactory comm = new ComplexCommandFactory();
+
+		ActionListener rectangle = new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e) {
+				comm.drawRectangle(0,0, 50, 25).execute(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
+
+		ActionListener triangle = new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e)
+			{
+				comm.drawTriangle(-50,-50, 50, 50).execute(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
```

Comment:
It is good to keep a convention used in the original code (class instance instead of robust anonymous functions)